Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -1082,20 +1098,10 @@ def _id_set_tests_matching_marketplace_value(self) -> Optional[CollectionResult]
 
         return CollectionResult.union(result)
 
-    def _collect_all_marketplace_compatible_packs(self) -> Optional[CollectionResult]:
-        result = []
-        for pack_metadata in PACK_MANAGER.iter_pack_metadata():
-            try:
-                result.append(self._collect_pack(
-                    pack_id=pack_metadata.pack_id,
-                    reason=CollectionReason.PACK_MARKETPLACE_VERSION_VALUE,
-                    reason_description=self.marketplace.value,
-                    allow_incompatible_marketplace=False,
-                    is_nightly=True,
-                ))
-            except (NothingToCollectException, NonXsoarSupportedPackException) as e:
-                logger.debug(str(e))
-        return CollectionResult.union(result)
+
+class UploadAllCollector(TestCollector):
```

Comment:
doesn't this one have a special `collect()` behavior? Removing tests or something?

Reply:
Need to ask @yaakovpraisler.

Reply:
@dorschw - it seems that the `_collect_all_marketplace_compatible_packs` function collects only packs, so no need to remove the tests as in UploadBranchCollector